README.md

```diff
@@ -58,6 +58,12 @@ CarrierWave.configure do |config|
     secret_access_key: ENV.fetch('AWS_SECRET_ACCESS_KEY'),
     region:            ENV.fetch('AWS_REGION') # Required
   }
+
+  # Optional: Signing of download urls, e.g. for serving private
+  # content through CloudFront.
+  config.sign_urls = {
```

Comment:
Configuration is all injected into the `CarrierWave::Uploader::Base` namespace. Any `aws` specific configuration should be prefixed with `aws_` to prevent collisions with other gems.